common/views/components/NewsletterPromo/NewsletterPromo.js

```diff
@@ -244,7 +238,23 @@ const NewsletterPromo = () => {
                       />
                       <input type="hidden" name="addressbookid" value="40131" />
                       <FormElementWrapper>
-                        <NewsletterInput />
+                        <TextInput
+                          required={true}
+                          id={'newsletter-input'}
+                          type={'email'}
+                          name={'email'}
+                          label={'Your email address'}
+                          errorMessage={
+                            isSubmitError
+                              ? 'There was a problem. Please try again.'
+                              : 'Enter a valid email address.'
+                          }
+                          value={value}
+                          handleChange={event =>
+                            setValue(event.currentTarget.value)
```

Comment:
You could probably set this as the default for inputs.

Reply:
yeah, good point